src/vs/editor/contrib/inlineCompletions/browser/inlineCompletionsAccessibleView.ts

```diff
@@ -17,6 +17,7 @@ export class InlineCompletionsAccessibleView extends Disposable implements IAcce
 	readonly priority = 95;
 	readonly name = 'inline-completions';
 	readonly when = ContextKeyExpr.and(InlineCompletionContextKeys.inlineSuggestionVisible);
+	private _provider: AccessibleContentProvider | undefined;
```

Comment:
Somehow the comment got lost, but I was curious why _provider needs to be stashed on these implementations. From what I see it's up to the consumer to dispose of it so there's no need retain it. If you do need to retain it maybe using a MutableDisposable is better so that the old providers are automatically disposed of when the new one is created and assigned.

Reply:
there were tons of implementations, it appears I just missed this one 😅

Reply:
we don't need to retain this

Reply:
looks like I need to remove this in an additional 5 places, will do